fonts.json

```diff
@@ -648,7 +648,7 @@
             "semi-narrow"
         ],
         "website": "https://github.com/rbanffy/3270font",
-        "year": 2015
+        "year": 2022
```

Comment:
The year here represents the first publication of the font, like the “birth year”, so it should stay at 2015.

Thanks for the update!

Reply:
Oh... OK. Let me fix it and do a push. Thanks for the prompt response, BTW!